File: src/app/src/main/java/com/example/screenservicetvapp/datamodels/LayoutTemplatePropertyDataModel.java
package com.example.screenservicetvapp.datamodels;

import com.google.gson.annotations.SerializedName;

public class LayoutTemplatePropertyDataModel {
    @SerializedName("key")
    private String key;
    @SerializedName("value")
    private String value;
    @SerializedName("label")
    private String label;

    public String getKey() {
        return key;
    }

    public String getValue() {
        return value;
    }
}

File: src/app/src/main/java/com/onscreensync/tvapp/datamodels/LayoutTemplatePropertyDataModel.kt
package com.onscreensync.tvapp.datamodels

import com.google.gson.annotations.SerializedName

data class LayoutTemplatePropertyDataModel(
    @SerializedName("key")
    val key: String?,
    @SerializedName("value")
    val value: String?,
    @SerializedName("label")
    val label: String?
)
